packages/embark-library-manager/src/index.js

```diff
@@ -39,6 +41,63 @@ class LibraryManager {
     });
   }
 
+  installAll(callback) {
+    const installAll = async () => {
```

Comment:
Any particular reason this function is (re)defined every time `installAll()` is called? Maybe we can make it a method instead?

Reply:
We could define an `_installAll` method and have this one call that one. But since the call is wrapped with `callbackify` it would have to be:

```
callbackify(this._installAll.bind(this))(callback)
```

[function.bind](https://devdocs.io/javascript/global_objects/function/bind) creates a new function every time; there are a couple of other strategies but all boil down to preserving `this` when wrapped with `callbackify` and so will also involve creating some function every time.

The function only gets called once, when an `embark [cmd]` is starting up, so I don't think there's a performance penalty to worry about.